### ISSUE TITLE
'showNavigationIndicators' does not exist on type 'NgbCarouselConfig'

### ISSUE THREAD

**Issue by sahilkatia** [ng-bootstrap/ng-bootstrap#2710]
```
 constructor(carouselconfig: NgbCarouselConfig) {
    this._selectedIndex = 0;
    carouselconfig.showNavigationIndicators = false; // its not recognizing this
    carouselconfig.keyboard = true// its recognizing this 
  }
```

**Comment by maxokorokov** [comment]
Please use the issue template: versions, stackblitz, etc.

Note that it's available in carousel from 2.2.0 only:
```ts
  /**
   * A flag to show / hide navigation indicators.
   * @since 2.2.0
   */
  @Input() showNavigationIndicators: boolean;
```

- if you believe that we've got a bug in this library than please provide a minimal reproduce scenario in a stackblitz or plunker: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/README.md#you-think-youve-found-a-bug
- if you are unsure if this is a bug or not than please use an appropriate support channel: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/README.md#getting-help

**Comment by sahilkatia** [comment]
I am working with "@ng-bootstrap/ng-bootstrap": "^1.0.4", on Angular5 thats the reason its not showing up?

**Comment by pkozlowski-opensource** [comment]
This functionality is only available from 2.2.0, you will have to upgrade.

**Comment by HillenLin** [comment]
I have the same problem. 
In the document, It is saying Angular 5 only support ng-bootstrap 1.X.X. Does this mean I can't use showNavigationIndicators in my Angular 5 project? In terms of upgrading? how to ? 
When I reinstall the package, it shows: 
npm WARN ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-bootstrap@3.0.1 requires a peer of @angular/common@>=6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-bootstrap@3.0.1 requires a peer of @angular/core@>=6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
